shared/redux/actions/eth.js

```diff
@@ -80,14 +80,22 @@ const getTransaction = () =>
       })
   })
 
-const send = (from, to, amount) =>
+const send = (from, to, amount, { gasPrice, gasLimit } = {}) =>
   new Promise(async (resolve, reject) => {
-    const { user: { ethData: { privateKey } } } = getState()
+    const { user: { ethData: { privateKey, gasRate } } } = getState()
+
+    if (!gasPrice) {
```

Comment:
```suggestion
      gasPrice = gasPrice || gasRate.price.normal
```

можно вот так делать, чуть короче получается

Reply:
Круто, спасибо